app/screens/gallery/video_renderer/index.tsx

```diff
@@ -59,13 +68,16 @@ const VideoRenderer = ({height, index, initialIndex, item, isPageActive, onShoul
     const serverUrl = useServerUrl();
     const videoRef = useAnimatedRef<Video>();
     const showControls = useRef(!(initialIndex === index));
+    const captionsEnabled = useContext(CaptionsEnabledContext);
     const [paused, setPaused] = useState(!(initialIndex === index));
     const [videoReady, setVideoReady] = useState(false);
     const [videoUri, setVideoUri] = useState(item.uri);
     const [downloading, setDownloading] = useState(false);
     const [hasError, setHasError] = useState(false);
     const source = useMemo(() => ({uri: videoUri}), [videoUri]);
 
+    const {track, selected} = getTranscriptionUri(serverUrl, item.postProps);
```

Comment:
should you use here `useMemo`? this function returns a new object every time this component updates for whatever reason, play/pause etc.. is there a reason why we need to re-run this function?